test/test_stats.py

```diff
@@ -14,13 +14,19 @@ def test_list_from_string(self):
         output = fixture("diff_numstat").decode(defenc)
         stats = Stats._list_from_string(self.rorepo, output)
 
-        self.assertEqual(2, stats.total["files"])
-        self.assertEqual(52, stats.total["lines"])
-        self.assertEqual(29, stats.total["insertions"])
+        self.assertEqual(3, stats.total["files"])
```

Comment:
Can you help me understand why this amount (and the following two) changed?

Reply:
I added another `diff_numstat file with change type `A` just to check. The other two files `a.txt` and `b.txt` have either deletions or changes and are of change type `C`, `b.txt` is potentially `R` but not sure about that.
Anyway if there will be a more dynamic testing with really adding, removing, changing files in a fake repo it would be good to not only test for `M`, so this could be a starting point

Reply:
Ah, right, this adds up perfectly. Apologies for not seeing it earlier.